scripts/populate_institutions.py

```diff
@@ -486,7 +498,7 @@ def main(env):
                 'logo_name': 'brown-shield.png',
                 'login_url': SHIBBOLETH_SP_LOGIN.format(encode_uri_component('https://sso.brown.edu/idp/shibboleth')),
                 'logout_url': SHIBBOLETH_SP_LOGOUT.format(encode_uri_component('https://test.osf.io/goodbye')),
-                'domains': ['test-osf.brown.edu'],
+                'domains': ['test-osf-brown.cos.io'],
```

Comment:
@sloria This ticket also updates all `test` institutions who has `domains` to `test-osf-<id>.cos.io` to be consistent with the others.